l2tdevtools/dpkg_files.py

```diff
@@ -145,10 +145,6 @@ class DPKGBuildFilesGenerator(object):
   _COPYRIGHT_TEMPLATE = '\n'.join([
       ''])
 
-  _INSTALL_TEMPLATE_PYTHON_TOOLS = '\n'.join([
```

Comment:
I would need to have a closer look at this. I opt to undo the remove for now and add a todo.
https://github.com/log2timeline/l2tdevtools/issues/620

Reply:
done